modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -18,4 +18,82 @@ Depending on your {ibm-z-name} environment, you can choose from the following op
+====
+
+[id="networking-reqs-ibmz_{context}"]
+==  Networking requirements for {ibm-z-title}
+
+In {ibm-z} environments, advanced networking technologies like Original Storage Architecture (OSA), HiperSockets, and Remote Direct Memory Access (RDMA) over Converged Ethernet (RoCE) require specific configurations that deviate from the standard settings used in the Agent Based Installer deployments. These overrides are necessary to accommodate their unique requirements and ensure a successful and efficient deployment on {ibm-z}.
+
+The following table lists the network devices that are supported for the network configuration override functionality:
+
+[cols="3,2,2,2,2", options="header"]
+|====
+| Network device 
+| z/VM  
+| KVM 
+| LPAR Classic 
+| LPAR Dynamic Partition Manager (DPM)
+
+| Original Storage Architecture (OSA) virtual switch 
+| Not needed
+| --
+| Not needed
+| Not needed
+
+| Direct attached OSA 
+| Supported
+| Only through a Linux bridge 
+| Supported
+| Not required
+
+| RDMA over Converged Ethernet (RoCE) 
+| Not needed
+| Only through a Linux bridge 
+| Not needed
+| Not needed
+
+| HiperSockets 
+| Supported 
+| Only through a Linux bridge 
+| Supported 
+| Not needed 
+|====
+
+[id="configuring-network-overrides-ibmz_{context}"]
+=== Configuring network overrides in {ibm-z-title}
+
+You can specify a static IP address on {ibm-z} machines that uses Logical Partition (LPAR) and z/VM. This is specially useful when the network devices do not have a static MAC address assigned to them. 
+
+If you have an existing `.parm` file, edit it to include the following entry:
+
+[source,terminal]
+----
+ai.ip_cfg_override=1
+----
+
+This parameter allows the file to add the network settings to the CoreOS installer.
+
+.Example of the `.parm` file
+[source,terminal]
+----
+rd.neednet=1 cio_ignore=all,!condev
+console=ttysclp0
+coreos.live.rootfs_url=<coreos_url> <1>
+ip=<ip>::<gateway>:<netmask>:<hostname>::none nameserver=<dns> 
+rd.znet=qeth,<network_adaptor_range>,layer2=1
+rd.<disk_type>=<adapter> <2> 
+rd.zfcp=<adapter>,<wwpn>,<lun> random.trust_cpu=on <3>
+zfcp.allow_lun_scan=0 
+ai.ip_cfg_override=1
+ignition.firstboot ignition.platform.id=metal 
+random.trust_cpu=on
+----
+<1> For the `coreos.live.rootfs_url` artifact, specify the matching `rootfs` artifact for the `kernel` and `initramfs` that you are booting. Only HTTP and HTTPS protocols are supported.
+<2> For installations on direct access storage devices (DASD) type disks, use `rd.` to specify the DASD where {rhel-first} is to be installed. For installations on Fibre Channel Protocol (FCP) disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {rhel} is to be installed.
+<3> Specify values for `adapter`, `wwpn`, and `lun` as in the following example: `rd.zfcp=0.0.8002,0x500507630400d1e3,0x4000404600000000`.
```

Comment:
So there are a few things about this snippet that need addressing:

1. If this is supposed to be an example file, then the user replaced values such as <coreos_url> should be an actual value instead of a placeholder. If you cannot find example values in time for 4.17 GA, please make sure you address this in another PR as soon as you can.
2. The callout labels below the snippet don't seem appropriate to include for a step about adding the `ai.ip_cfg_override=1` parameter to the file, since this procedure does not talk about configuring things like the `coreos.live.rootfs_url` artifact and `rd.<disk_type>=<adapter>`. You should probably remove the callout labels from the snippet and remove these descriptions.